lib/src/chart/base/axis_chart/axis_chart_data.dart

```diff
@@ -230,8 +236,7 @@ class AxisTitles with EquatableMixin {
   bool get showAxisTitles => axisNameWidget != null && axisNameSize != 0;
 
   /// If there is something to show as sideTitles, it returns true
-  bool get showSideTitles =>
-      sideTitles.showTitles && sideTitles.reservedSize != 0;
+  bool get showSideTitles => sideTitles.showTitles && sideTitles.reservedSize != 0;
```

Comment:
Unnecessary change.

Reply:
Previously I used the darts code style line length to be 120 and forgot to change it to default (80) while working on this. So when I use auto wrapping, that's what happens. Please review my latest changes. Thankyou